WalletWasabi.Gui/Controls/WalletExplorer/SendTabViewModel.cs

```diff
@@ -891,6 +863,18 @@ private async Task DoDequeueAsync(IEnumerable<CoinViewModel> selectedCoins)
 			}
 		}
 
+		public string FeeText => FeeRate is null ? "" :
+										(FeeDisplayFormat == FeeDisplayFormat.SatoshiPerByte ? $"(~ {FeeRate.SatoshiPerByte} sat/byte)" :
+										 (FeeDisplayFormat == FeeDisplayFormat.USD ? $"(~ ${UsdFee.ToString("0.##")})" :
+										  (FeeDisplayFormat == FeeDisplayFormat.BTC ? $"(~ {EstimatedBtcFee.ToString(false, false)} BTC)" :
+										   (FeeDisplayFormat == FeeDisplayFormat.Percentage ? $"(~ {FeePercentage.ToString("0.#")} %)" : throw new NotSupportedException("This is impossible.")))));
+
+		public string FeeToolTip => FeeRate is null ? "" :
+										(FeeDisplayFormat == FeeDisplayFormat.SatoshiPerByte ? "Expected fee rate in satoshi / vbyte." :
+										 (FeeDisplayFormat == FeeDisplayFormat.USD ? $"Estimated total fees in USD. Exchange Rate: {(long)UsdExchangeRate} BTC/USD." :
+										  (FeeDisplayFormat == FeeDisplayFormat.BTC ? "Estimated total fees in BTC." :
+										   (FeeDisplayFormat == FeeDisplayFormat.Percentage ? "Expected percentage of fees against the amount to be sent." : throw new NotSupportedException("This is impossible.")))));
```

Comment:
Same as above.

Reply:
No, this is not how it should be done. In the MVVM pattern there is a clear separation of concerns:

* what internal info (data model) is needed for the functionality is in the `ViewModel` and, 
* what has to be displayed and how that has to be displayed is a `View` responsibility

In this case, the `ViewModel` shouldn't know that in the `View` is a `ToolTip` because that is a UI concept and not a model concept. The `ViewModel` should only be aware of the `FeeRate` and the user's formatting preference (`FeeDisplayFormat`). It is the `View` the component that has to know how to express the `ViewModel` as GUI controls.

This `FeeToolTip` property is a **converter** and as we know, those converters should not be in the `ViewModels` but in a `IConverter`.

---------------------------

*Mea culpa:* we are guilty of doing and abusing of this anti-pattern in the past.

Reply:
Same reply as above. (By @benthecarman )